the-blue-alliance-ios/Services/PushService.swift

```diff
@@ -61,6 +56,8 @@ class PushService: NSObject {
                     Crashlytics.sharedInstance().recordError(error)
                     self?.registerRetryable()
                 } else {
+                    // TODO: Make sure this timer gets removed from memory...
```

Comment:
Let's write a test for this or something - even if this is the only thing we test right now for the push service. Or, hell, do it manually. But confirm this before merging.

Reply:
I can't see that the timer gets removed from memory, but it seems like our retry service isn't holding on to it, and I can't find a reference in our memory graph where the timer is holding on to the push service soooo

Reply:
Yeah - it looks like the block referencing the push service gets properly deallocated when we invalidate the timer and whatnot. I'm not going to worry about why the timer is still in memory - that's the run loops problem at that point